src/lib/utils/is-object-empty.js

```diff
@@ -1,8 +1,12 @@
 export default function isObjectEmpty(obj) {
-    var k;
-    for (k in obj) {
-        // even if its not own property I'd still call it non-empty
-        return false;
+    if (Object.keys) {
+        return (Object.keys(obj).length === 0);
+    } else {
+        var k;
+        for (k in obj) {
+            // even if its not own property I'd still call it non-empty
```

Comment:
Object.keys only looks at own properties.  I don't personally think that behavior change is a deal breaker for this PR -- but I do think that we need to have consistent behavior in all browsers, so could you please do a `hasOwnProperty` check here?